build.sbt

```diff
@@ -67,69 +87,70 @@ lazy val moped = project.settings(
   libraryDependencies ++=
     List(
       "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-      "dev.dirs" % "directories" % "21",
-      "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.2",
-      "io.github.alexarchambault" %% "data-class" % "0.2.5",
-      "com.lihaoyi" %% "os-lib" % "0.7.1",
-      "com.lihaoyi" %% "ujson" % "1.2.2",
-      "com.lihaoyi" %% "pprint" % "0.6.0",
-      "com.lihaoyi" %% "fansi" % "0.2.9",
-      "org.typelevel" %% "paiges-core" % "0.3.2"
+      "dev.dirs" % "directories" % V.dirs,
+      "org.scala-lang.modules" %% "scala-collection-compat" %
+        V.collectionCompat,
+      "io.github.alexarchambault" %% "data-class" % V.dataClass,
+      "com.lihaoyi" %% "os-lib" % V.osLib,
+      "com.lihaoyi" %% "ujson" % V.ujson,
+      "com.lihaoyi" %% "pprint" % V.pprint,
+      "com.lihaoyi" %% "fansi" % V.fansi,
+      "org.typelevel" %% "paiges-core" % V.paiges
     ),
-  crossScalaVersions := List(scala212, scala213)
+  crossScalaVersions := List(V.scala212, V.scala213)
 )
 
 lazy val hocon = project
   .in(file("moped-hocon"))
   .settings(
     moduleName := "moped-hocon",
-    libraryDependencies ++= List("org.ekrich" %% "sconfig" % "1.3.4"),
-    crossScalaVersions := List(scala212, scala213)
+    libraryDependencies ++= List("org.ekrich" %% "sconfig" % V.sconfig),
+    crossScalaVersions := List(V.scala212, V.scala213)
   )
   .dependsOn(moped)
 
 lazy val yaml = project
   .in(file("moped-yaml"))
   .settings(
     moduleName := "moped-yaml",
-    libraryDependencies ++= List("org.yaml" % "snakeyaml" % "1.26"),
-    crossScalaVersions := List(scala212, scala213)
+    libraryDependencies ++= List("org.yaml" % "snakeyaml" % V.snakeyaml),
+    crossScalaVersions := List(V.scala212, V.scala213)
   )
   .dependsOn(moped)
 
 lazy val toml = project
   .in(file("moped-toml"))
   .settings(
     moduleName := "moped-toml",
-    libraryDependencies ++= List("tech.sparse" %% "toml-scala" % "0.2.2"),
-    crossScalaVersions := List(scala212, scala213)
+    libraryDependencies ++= List("tech.sparse" %% "toml-scala" % V.tomlScala),
+    crossScalaVersions := List(V.scala212, V.scala213)
   )
   .dependsOn(moped)
 
 lazy val dhall = project
   .in(file("moped-dhall"))
   .settings(
     moduleName := "moped-dhall",
-    libraryDependencies ++= List("org.dhallj" %% "dhall-scala" % "0.7.0-M1"),
-    crossScalaVersions := List(scala212, scala213)
+    libraryDependencies ++= List("org.dhallj" %% "dhall-scala" % V.dhallScala),
+    crossScalaVersions := List(V.scala212, V.scala213)
   )
   .dependsOn(moped)
 
-lazy val jsonnet = project
+/*lazy val jsonnet = project
```

Comment:
commenting out for now, as there's been an idea floated to publish sjsonnet module under com-lihaoyi ecosystem

Reply:
I would just remove it, we can get back to it easily